src/service/action-impl.js

```diff
@@ -271,9 +273,17 @@ export class ActionService {
       return;
     }
 
+    // Check if this is an element has a known action.
+    const supportedActions = ELEMENTS_ACTIONS_MAP_[lowerTagName];
+    if (target[ACTION_QUEUE_] && supportedActions &&
```

Comment:
Why are you not initializing `ACTION_QUEUE_` right here? Are risk conditions do not apply here? Is it even worth the cost?

Btw, this `if` and the previous one (`target.id ~= amp-`) can be combined.